custom_components/solarfocus/sensor.py

```diff
@@ -519,6 +520,7 @@ def native_value(self):
     SensorEntityDescription(
         key="message_number",
         icon="mdi:message-text-outline",
+        device_class="solarfocus_pbmessagenumber"
```

Comment:
The way translations work changed in 2023.4. A translation key will be automatically generated 
https://github.com/LavermanJJ/home-assistant-solarfocus/blob/6140137d6f7c6da526efae0d66493185594d042c/custom_components/solarfocus/entity.py#L58-L67

-> bo_message_number would be the translation key to use then in the `strings.json` file 
 e.g. 
https://github.com/LavermanJJ/home-assistant-solarfocus/blob/6140137d6f7c6da526efae0d66493185594d042c/custom_components/solarfocus/strings.json#L63

And also in translations/en.json and translations/de.json